python/lsst/sconsUtils/state.py

```diff
@@ -196,6 +198,25 @@ def _initEnvironment():
         #
         env['LIBDIRSUFFIX'] = '/'
 
+    if 'SCONSUTILS_USE_CONDA_COMPILERS' in os.environ:
```

Comment:
```suggestion
    if 'SCONSUTILS_USE_CONDA_COMPILERS' in os.environ:
        # Pulling in conda-build flags from the environment
```

Reply:
I'd say more accurately this is pulling in the flags for the conda compilers. Those flags are defined not within conda build but in conda packages that have activation scripts.